tests/garage/torch/algos/test_pearl.py

```diff
@@ -21,24 +22,25 @@
 class TestPEARL:
     """Test class for PEARL."""
 
+    @pytest.mark.large
```

Comment:
@lywong92 Given that you have an example `pearl_ml1_push.py`, I think this test here is doing the same thing, so it is not necessary. Am I correct?

Reply:
We don't count examples towards test coverage though, so keep that in mind

Reply:
@naeioi I think `pearl_ml1_push.py` is added to `LONG_RUNNING_EXAMPLES`